vnext/Microsoft.ReactNative/ReactRootView.idl

```diff
@@ -36,5 +36,8 @@ namespace Microsoft.ReactNative
 
     DOC_STRING("Reloads the current @ReactRootView UI components.")
     void ReloadView();
+
+    DOC_STRING("Use Fabric for this ReactRootView")
+    Boolean ExperimentalUseFabric { get; set; };
```

Comment:
Inevitably there will be people who try to enable this if they see it in their Intellisense😂. I guess it won't do anything without the Quirk setting and compile-time flag though, but maybe that could be confusing. Not a huge issue if it's marked as experimental though.

Maybe overkill but we could probably hide the property in a different interface that we can QI to if we don't want people to see this in autocomplete or whatever.

Reply:
can you please mark these as [experimental]? If not allowed on properties, can you put these on an interface that's marked [experimental]?